lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -25,10 +25,24 @@ const axe = fs.readFileSync(
 );
 
 // This is run in the page, not Lighthouse itself.
+// axe.run returns a promise which fulfills with a results object
+// containing any violations.
 /* istanbul ignore next */
 function runA11yChecks() {
-  return new Promise((resolve, reject) => {
-    axe.a11yCheck(document, resolve);
+  return axe.run(document, {
+    runOnly: {
+      type: 'rule',
+      values: [
+        'aria-allowed-attr',
```

Comment:
is this list worth automating? It's relatively small right now, but will require us to (remember to) update it every time there's a new audit. There's already a lot of things to check off the list when adding a new audit.